app/controllers/mail_notification_controller.rb

```diff
@@ -2,16 +2,17 @@
 
 class MailNotificationController < ApplicationController
   skip_before_action :require_active_user_login, raise: false
+  layout 'not_logged_in'
 
-  def update
-    user = User.find_by(unsubscribe_email_token: params[:token])
+  def confirm
+    @user = User.find_by(unsubscribe_email_token: params[:token])
```

Comment:
変数のスコープの話で、`@user`ではなく、`user`を使った方が良いかなと思いました！
この変数は`confirm`メソッドの条件分岐でしか使わないので、インスタンス変数にする必要はない気がします👀
```suggestion
    user = User.find_by(unsubscribe_email_token: params[:token])
```

Reply:
`@user`は`app/views/mail_notification/confirm.html.slim:13`で使われているのでそのままにしました！
（`user`にするとその箇所でエラーになります）
ご確認をお願いいたします！

Reply:
あ、そうですね💦失礼しました🙇
`user`にして確認もしたところ、エラーになりました🙆‍♀️